test/testinput/bufr_ncep_highRes_sonde.yaml

```diff
@@ -0,0 +1,167 @@
+# (C) Copyright 2020 NOAA/NWS/NCEP/EMC
+# 
+# This software is licensed under the terms of the Apache Licence Version 2.0
+# which can be obtained at http://www.apache.org/licenses/LICENSE-2.0.
+
+observations:
+  - obs space:
+      name: bufr
+
+      obsdatain: "./gdas.t12z.adpupa_nc002103.tm00.bufr_d"
+
+      exports:
+        subsets:
+          - NC002103
+        group_by_variable: pressure
+        variables:
+          # MetaData
+          timestamp:
+            datetime:
+              year: "*/YEAR"
+              month: "*/MNTH"
+              day: "*/DAYS"
+              hour: "*/HOUR"
+              minute: "*/MINU"
+          stationIdentification:
+            query: "*/RPID"
+          latitude:
+            query: "*/CLATH"
+          longitude:
+            query: "*/CLONH"
+          stationElevation:
+            query: "*/HSMSL"
+
+          # ObsValue
+          pressure:
+            query: "*/UARLVB/PRLC"
+          airTemperature:
+            query: "*/UARLVB/UATMP/TMDB"
+          dewpointTemperature:
+            query: "*/UARLVB/UATMP/TMDP"
+          windDirection:
+            query: "*/UARLVB/UAWND/WDIR"
+          windSpeed:
+            query: "*/UARLVB/UAWND/WSPD"
+
+          # ObsValue - ocean
+          waterTemperature:
+            query: "*/SST1"
+
+          # QualityMarker
+          stationElevationQM:
+            query: "*/QCEVR"
+          pressureQM:
+            query: "*/UARLVB/QMPR"
+          airTemperatureQM:
+            query: "*/UARLVB/UATMP/QMAT"
+          dewpointTemperatureQM:
+            query: "*/UARLVB/UATMP/QMDD"
+          windSpeedQM:
+            query: "*/UARLVB/UAWND/QMWN"
+
+
+    ioda:
+      backend: netcdf
+      obsdataout: "../testoutput/gdas.t12z.adpupa_bufr_nc002103.tm00.nc"
+
+      dimensions:
+        - name: RadiosondeReportLevelData
+          path: "*/UARLVB"
+
+      variables:
+        # MetaData
+        - name: "MetaData/dateTime"
+          coordinates: "longitude latitude"
+          source: variables/timestamp
+          longName: "Datetime"
+          units: "seconds since 1970-01-01T00:00:00Z"
+
+        - name: "MetaData/stationIdentification"
+          coordinates: "longitude latitude"
+          source: variables/stationIdentification
+          longName: "Station Identification"
+
+        - name: "MetaData/latitude"
+          coordinates: "longitude latitude"
+          source: variables/latitude
+          longName: "Latitude"
+          units: "degree_north"
+          range: [-90, 90]
+
+        - name: "MetaData/longitude"
+          coordinates: "longitude latitude"
+          source: variables/longitude
+          longName: "Longitude"
+          units: "degree_east"
+          range: [-180, 180]
+
+        - name: "MetaData/stationElevation"
+          coordinates: "longitude latitude"
+          source: variables/stationElevation
+          longName: "Elevation of Observing Location"
```

Comment:
It would seem to me a better long name is simply `Station Elevation` because elevation of the "observing location" would imply it is the height of the radiosonde along the observation path whereas the balloon was generally launched from a ground (or sea/mobile) station.  Alternatively, `Elevation of launch location` could be used, and, truthfully the long name isn't actually used in software, but still nice to get it stated as well as possible.

Reply:
@gthompsnJCSDA "Elevation of Observing Location" is changed to "Station Elevation".